### PR TITLE
Release 5.6.0

### PR DESCRIPTION
**Pull request by korshaknn**
Related to #1183

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ repositories {
 }
 
 dependencies {
-    implementation 'com.mapbox.mapboxsdk:mapbox-sdk-services:5.6.0-SNAPSHOT'
```
Comment (LukasPaczos) [inline README.md:49]: You also need to update the `gradle.properties` constant.

Comment (korshaknn) [reply README.md:49]: fixed

**Comment by korshaknn** [comment]
need to revert 1 commit from master. Will be created a new PR for release

**Comment by korshaknn** [comment]
need to revert 1 commit from master. Will be created a new PR for release

**Pull request closed without merging**